04_numeronym/sahidakhtar1/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	result := make([]string, len(vals))
+	for i, v := range vals {
+		result[i] = createNumeronym(v)
+	}
+	return result
+}
+
+func createNumeronym(s string) string {
+	trimedString := strings.TrimSpace(s)
+	var numeronym string
+	//If the string contains space or \t or \n then it will return the same string
+	if strings.Contains(trimedString, " ") ||
+		strings.Contains(trimedString, "\t") ||
+		strings.Contains(trimedString, "\n") {
+		return s
+	}
+	var count int
+	var s1, s2 string
```

Comment:
Better convert to `runes := []rune(trimedString)` and `runes[0]`, `runes[len(runes) - 1]`

Reply:
Done